stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/repository/domain/message/MessageRepository.kt

```diff
@@ -26,6 +26,7 @@ internal interface MessageRepository {
 internal class MessageRepositoryImpl(
     private val messageDao: MessageDao,
     private val getUser: suspend (userId: String) -> User,
+    private val getCurrentUser: suspend () -> User?,
```

Comment:
Why we can't pass just user from `RepositoryFacade`? Look at `RepositoryFacadeBuilder::currentUser`

Reply:
Yep, makes sense, will just make it nullable for noop repos.